src/Link.cc

```diff
@@ -190,6 +189,23 @@ std::optional<math::Pose3d> Link::WorldPose(
       .value_or(sim::worldPose(this->dataPtr->id, _ecm));
 }
 
+//////////////////////////////////////////////////
+std::optional<math::Inertiald> Link::WorldInertial(
```

Comment:
include `<optional>` and `math::Inertiald`

Reply:
This are being included in the `link.hh` file, should I move them to this file?

Reply:
I don't think that's necessary.